classes/requests/helpers/class-truelayer-helper-order.php

```diff
@@ -43,4 +43,17 @@ public static function get_account_holder_name( $order ) {
 
 		return $account_holder_name;
 	}
+
+    /**
+     * Get order meta - user date of birth
+     * @param $order
+     * @return mixed
+     */
+    public static function get_user_date_of_birth( $order ) {
+
+        $birth_date = apply_filters('truelayer_birth_date_field', $order->get_meta( '_truelayer_user_birth_date', true ), $order->get_id(), $order);
```

Comment:
I think that we should do a small tweak here. It can be done in 2 ways.

1.  Change the name of the filter `truelayer_birth_date_field` to `truelayer_date_of_birth`. This is because the data in the filter is the actual date of birth, not the field name.
2. Keep the name `truelayer_birth_date_field` but let the filter be for the actual meta data name. Like this: 

```
$truelayer_birth_date_field = apply_filters('truelayer_birth_date_field', '_truelayer_user_birth_date' , $order->get_id(), $order);
$birth_date = $order->get_meta( $truelayer_birth_date_field '_truelayer_user_birth_date', true );
```

I guess this could be done in a singel line logic as well if wanted :).

Reply:
@NiklasHogefjord I tweaked date of birth field. 
Couldn't find better solution for country problem. Wrote in channel.